modules/configuration/test/ConfigurationTest.php

```diff
@@ -155,6 +157,8 @@ public function testAllLinks()
     {
 
         $this->safeGet($this->url . "/configuration/");
+        // wait for the React to render
```

Comment:
Please replace this function in this file. using safeFindelement replace findElements 
    private function _linkTest($text)
    {
        $this->safeClick(WebDriverBy::linkText($text));
	$webActives =  $this->safeFindElement(WebDriverBy::cssSelector(".active"));
        $bodyText   = $webActives->getText();
        $this->assertStringContainsString($text, $bodyText);
    }

Reply:
Regarding the testplan3 in the dashboard test, you can temporarily comment it out. I've identified a solution to address this issue, and I'll implement it later. https://github.com/kongtiaowang/Loris/pull/381/files#diff-2af5e853d357c98c23e22e3a7d855bc560485103cb82026fa408d7fdf6f3fa5d